wand/image.py

```diff
@@ -1028,6 +1110,51 @@ def background_color(self, color):
             if not result:
                 self.raise_exception()
 
+    def matte(self, flag):
```

Comment:
It seems write-only, and it is the reason why it’s not a `property`, isn’t it?  I think it would be better if the method name is `set_matte()`.